services/app-web/src/gqlHelpers/mutationWrappersForUserFriendlyErrors.ts

```diff
@@ -36,26 +36,37 @@ const handleApolloErrorsAndAddUserFacingMessages = (
             ? ERROR_MESSAGES.submit_error_generic
             : ERROR_MESSAGES.unlock_error_generic
 
+    const options = {
+        cause: {},
```

Comment:
what is the cause option for an Error? Looks like we are translating the GQL ‘code’ into an Error cause?

Reply:
> Looks like we are translating the GQL ‘code’ into an Error cause?

Yep, I needed a way to know what kind of error it was. Specifically, an `EMAIL_ERROR`, in order to let the user complete the submission/unlock. Using the `cause` option seemed like a good place to put error codes. I guess I could have made it `cause.code`?

@haworku can clarify this, but I think `handleApolloErrorsAndAddUserFacingMessages` is returning `new Error` instead of `new ApolloError` or `new GraphQLError` because the server can return `Error` or `ApolloError` so this is handling it by converting all to `Error`?

There is a better way, but I wanted to leave that work till after the spike in server-side errors.